html_css/project_positioning.md

```diff
@@ -4,6 +4,8 @@ The New York Times has been a leader in experimenting with the inclusion of mult
 
 In this project, your assignment is to copy [this article from the New York Times](http://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)(open in an "incognito" window without cookies to view it if you're over your monthly limit).  Notice how the section titled "The Theory of Inflation" (you may need to widen your viewing window for it to show up) is laid out.
 
+You can use any or all of the positioning techniques you've learned in this section. The modern way would be to use Flexbox or Grid but it's perfectly fine if you want to float or use positioning for some or all of the project.
+
```

Comment:
You can use any or all of the positioning techniques you've learned in this section. The modern way would be to use Flexbox or Grid. There are certain times where FlexBox and Grid can't achieve the layout you want, knowing floats and other positioning techniques is valuable practice and you should try and use a combination of techniques for this project.

Reply:
Updated 👍